tests/dummy/app/styles/defaults/_examples.scss

```diff
@@ -0,0 +1,42 @@
+@import '../variables';
+
+.example {
+  centered {
+    background-color: $theme-color-1;
+    &:nth-child(even) {
+      background-color: $theme-color-2;
+    }
+  }
+}
+
+hbox.header {
+  height: 50px;
+}
+
+.blue {
+  background-color: $theme-color-1;
+}
+
+.fixed-250 {
+  width: 250px;
```

Comment:
this should be switched to `max-width` to play nice with flexbox. Also, all the elements which have this class should drop flexi-fit, else max-width is ignored